src/domain/interfaces/account.datasource.interface.ts

```diff
@@ -38,7 +41,7 @@ export interface IEmailDataSource {
    * @param args.code - the generated code to be used to verify this email address
    * @param args.verificationGeneratedOn – the date which represents when the code was generated
    */
-  saveEmail(args: {
+  saveAccount(args: {
```

Comment:
As above, I think this function documentation needs to be changed (from the concept `email` to the `account` one).

Reply:
[fa63a4e](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/fa63a4e9950036c1b044c936774359b805c7c62f)